src/mlpack/tests/metric_test.cpp

```diff
@@ -164,7 +164,7 @@ TEST_CASE("NMSMetricTest", "[MetricTest]")
   // Selected indices of bounding boxes using
   // torchvision.ops.nms().
   desiredIndices = arma::ucolvec(2);
-  desiredIndices << 0 << 2;
+  desiredIndices = { 0, 2 };
```

Comment:
Looks like we can remove the line above.

Reply:
Done.